src/components/Pages/LoginPage/LoginPage.tsx

```diff
@@ -99,12 +99,8 @@ export const LoginPage = () => {
           setConnectionModalState(ConnectionModalState.CONNECTING_WALLET)
           const connectionData = await connectToProvider(connectionType)
 
-          // The requests sign in flow for the desktop app has a different identity.
-          // There is no need to create one here if the user is coming from the requests page.
-          if (searchParams.get('fromRequests') !== 'true') {
```

Comment:
Removed at least for now given that with the avatar setup, in order to create a profile the identity is required. By doing this, the user would not have an identity when coming from the desktop client.
It's kind of annoying but I'll create an issue for a better solution.